_templates/exchanges.html

```diff
@@ -91,6 +91,7 @@ <h3 id="japan" class="no_toc">Japan</h3>
           <a class="marketplace-link" href="https://bitflyer.com/ja-jp/">bitFlyer</a>
           <br>
           <a class="marketplace-link" href="https://www.btcbox.co.jp/">BtcBox</a>
+          <a class="marketplace-link" href="https://coincheck.com/">Coincheck</a>
```

Comment:
Please add br before this code.

Reply:
Thank you for the comment. Fixing completed.